composer.json

```diff
@@ -25,7 +25,8 @@
     "typo3/cms-extbase": "^11.5",
     "typo3/cms-fluid": "^11.5",
     "typo3/cms-frontend": "^11.5",
-    "parsecsv/php-parsecsv": "^1.3"
+    "parsecsv/php-parsecsv": "^1.3",
+    "google/recaptcha": "^1.2"
```

Comment:
This should be in the require dev section and not included by default. If this validator is used in a project, it must be loaded there.

And do we still need mblaschke/jm-recaptcha after this? if nor remove it please.